src/components/field-key/qr-panel.vue

```diff
@@ -70,26 +70,31 @@ import DocLink from '../doc-link.vue';
 
 import FieldKey from '../../presenters/field-key';
 import { apiPaths } from '../../util/request';
+import { requestData } from '../../store/modules/request';
 
 export default {
   name: 'FieldKeyQrPanel',
   components: { CollectQr, DocLink },
   props: {
-    fieldKey: FieldKey, // eslint-disable-line vue/require-default-prop
-    managed: {
-      type: Boolean,
-      default: false
-    }
```

Comment:
The linter no longer flags `vue/require-default-prop`, so this can be simplified.